tests/unit/test_cube.py

```diff
@@ -47,6 +49,34 @@ def but_it_raises_on_other_cube_response_types(
 
         assert str(exception) == expected_value
 
+    def it_knows_if_it_is_mr_by_itself_to_help(
```

Comment:
The `_to_help` suffix is used to indicate a private (helper) method/property. Since this is a public method it is not needed here.